fcrepo-http-api/src/main/java/org/fcrepo/http/api/Transactions.java

```diff
@@ -84,19 +151,20 @@ public Response createTransaction() throws URISyntaxException {
      * @return 204
      */
     @PUT
-    @Path("/fcr:tx/{transactionId}/commit")
+    @Path("{transactionId}/commit")
     public Response commit(@PathParam("transactionId") final String txId) {
         try {
             final Transaction transaction = txManager.get(txId);
-            LOGGER.info("Commit transaction '{}'", transaction.getId());
+            LOGGER.info("Committing transaction '{}'", transaction.getId());
             transaction.commit();
             return noContent().build();
-        } catch(Exception e) {
-            if (e.getMessage().matches("No Transaction found with transactionId")) {
-                return status(BAD_REQUEST).entity(e.getMessage()).type(TEXT_PLAIN_WITH_CHARSET).build();
-            } else {
-                throw new RuntimeException(e);
-            }
+        } catch (final TransactionNotFoundException e) {
+            return Response.status(Status.BAD_REQUEST).build();
```

Comment:
Do we want to return  a text message as well such as "Transaction does not exist: {txId}"

Reply:
Or at least pass e.getMessage()